### ISSUE TITLE
Tutorial for building on windows -- help team?

### ISSUE THREAD

**Issue by epruesse** [conda-forge/staged-recipes#2856]
I've just tried to get `xorg-libxp` #2607 to work on win and run into a log train of issues. I'm giving up as it looks like it needs patching, deferring the task to the feedstock in accordance with @pkgw .

I'm guessing a significant share of the contributors and maintainers will lack experience building with mingw. Yet, having the majority of packages also work on that target would be beneficial to conda-forge. 

Are there volunteers willing to form a conda-forge/win-help team? And perhaps write up some best practices, faq or tutorial on how to get things working on that platform? The msym2 packages are rather particular to use.

It would be very helpful to have a way to set up a build environment on Linux/OSX in which the build can be emulated. Trial and error on AppVeyor is slow and tedious and also clogs up the build queue.

**Comment by pkgw** [comment]
@epruesse, agreed that building on Window can be very tricky if one is coming from a Linux/OSX background. It would indeed be very useful to assemble some materials giving tips.

One tip that was very helpful for me with the X.org stuff — there are a few places where Microsoft makes available free Windows VM images, so you can boot up a demo Windows machine and try out development even if you don't have a dedicated Windows box.

**Comment by stadelmanma** [comment]
I found it beneficial to just add Appveyor CI to complement Travis on my project, you get much quicker builds on your own pipeline and at least for me I found a few corner cases where things broke on Windows but not on Linux/OSX.

The one thing that would have saved me several dead-end CI runs would be snippets that suggest how to configure the widows path so you can access tools in MinGW directly. i.e.  `set PATH=%PATH%;C:\mingw-w64\x86_64-6.3.0-posix-seh-rt_v5-rev1\mingw64\bin`

**Comment by rolando** [comment]
Projects that support windows often run on appveyor too. So also you can check how they build the package as a starting point.

**Comment by jakirkham** [comment]
Should this issue be moved to the [`xorg-libxp` feedstock]( https://github.com/conda-forge/xorg-libxp-feedstock )?

**Comment by stadelmanma** [comment]
I would say the issue itself is scoped outside of the feedstock itself. I could draft a PR for a windows specific bullet point or two into the FAQ if you'd like. If you don't feel the need to add any Window's specific notes then I'd just close the issue.

**Comment by epruesse** [comment]
Yes, it's outside the feedstock scope. Feel free to close it.

Perhaps it would be useful to have a conda-forge/windows team of knowledgeable volunteers willing to help with win issues to have someone to @ ping.

**Comment by jakirkham** [comment]
We do have help teams. Maybe we should have one for Windows. It has come before. @CJ-Wright, thoughts?

**Comment by CJ-Wright** [comment]
It would be good to advertise those help teams as well.
